R/sparklyr.sedona/DESCRIPTION

```diff
@@ -0,0 +1,31 @@
+Type: Package
+Package: sparklyr.sedona
+Title: Sparklyr Extension for Apache Sedona
+Version: 0.1.0
+Authors@R:
+    c(person(family = "Apache Sedona",
+             role = c("aut", "cre"),
+             email = "dev@sedona.apache.org"),
```

Comment:
Oh I just realized the dev@s.a.o is open to the public and anyone has access to it. Would you please change this to private@sedona.apache.org? This way, only PMC members can control the publish of this doc. Once you are added to Sedona PMC (after voting), you will have access to private@sedona.apache.org.